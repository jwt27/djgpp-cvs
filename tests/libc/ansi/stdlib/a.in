../../crt0/args aaaaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbbbbbb ccccccccccccccccccccc ddddddddddddddddddddddd eeeeeeeeeeeeeeeeeeeee ffffffffffffffffffffffff gggggggggggggggggggg hhhhhhhhhhhhhhhhhhhhh iiiiiiiiiiiiiiiiii > a.out
