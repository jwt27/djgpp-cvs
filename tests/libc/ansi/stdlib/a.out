[0] = `d:/posix/tests/libc/ansi/stdlib/../../crt0/args.exe'
[1] = `aaaaaaaaaaaaaaaaaaaaaa'
[2] = `bbbbbbbbbbbbbbbbbbbbbbbb'
[3] = `ccccccccccccccccccccc'
[4] = `ddddddddddddddddddddddd'
[5] = `eeeeeeeeeeee'
